### DISCUSSION TITLE
How to access current game context from UI Context?

### DISCUSSION THREAD

**Discussion by realSamy** [efroemling/ballistica#255] [Q&A] [answered]
Hello there, I'm studying Bombsquad source code and docs and want to mod it to host my own mods games!
For the first step, I'm going to enable and disable slow mode from party chat, when I try to do something like:
```python3
_ba.getactivity().globalsnode.slow_motion = False # or True
```
inside the `ba._hooks.filter_chat_message` function, I get an error about wrong or non exists activity because I'm inside `UI Context`, How can I access the current playing game activity?

**Comment by Dliwk** [comment]
Try replacing `getactivity()` with something like `get_foreground_host_activity()`.

**Reply by realSamy** [reply]
thanks

**Comment by ghost** [comment, marked as answer]
Simple solution:
`activity = _ba.get_foreground_host_activity()`
`with ba.Context(activity): activity.globalsnode.slow_motion = False`

**Reply by realSamy** [reply]
thanks
